### DISCUSSION TITLE
2008/04/19/%E3%80%8A%E6%98%93%E4%B8%AD%E5%A4%A9%E5%93%81%E4%B8%89%E5%9B%BD%E3%80%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#217] [Announcements]
# 2008/04/19/%E3%80%8A%E6%98%93%E4%B8%AD%E5%A4%A9%E5%93%81%E4%B8%89%E5%9B%BD%E3%80%8B

今天终于听完了全套的《易中天品三国》，这是我最喜欢的《百家讲坛》专题之一。喜欢它的原因之一是我非常欣赏易中天先生的历史观：分析评价一个历史人物的 时候应该避免做道德评价，而是进行科学分析。换句话说，研究历史是为了以史为鉴，吸取前人的经验教训。因此，就不应该把一个人历史上的功过归结为其道德的 善恶，而是更应该关注社会制度对人的影响。

https://blog.qizhen.xyz/2008/04/19/%E3%80%8A%E6%98%93%E4%B8%AD%E5%A4%A9%E5%93%81%E4%B8%89%E5%9B%BD%E3%80%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

杰
2008年04月24日 18:59 编辑
小阮，当心回中国有人找你喝茶。
这些东西，我觉得，能够独立思考的都能明白，因为实在是太简单的真理，只是我不说。

回复
永青
2008年04月21日 11:16 编辑
光靠道德是没有用的，还是要制度和法律。可惜中国向来没有“法制”，先秦的法家充其量也只是“律制”。。
建议可读易中天《帝国的惆怅》、《帝国的终结》。

回复
竹贤
2008年04月20日 08:14 编辑
你小时候受到的那种政治教育正是反映了中国思想文化教育的一个重大失误，其影响是长期的。中国的几千年的文明史都是“成者为王败者寇”，历朝历代均如此。正因为如此，当我在二三十年前，读到林肯的《葛底斯堡演说》时，被强烈地震撼了。
就中国的道德观念和文化传统而言，看一件事，往往仅仅注重于“动机”和“效果”。而不知，“动机”他人是不可知的，只是他人的猜测和分析；而效果，也往往受到政治和人为主观因素的主导。也就是说，历史也可以由人为来装扮。因此，程序和过程的合法和合理更为重要。

回复
